code/modules/mob/living/silicon/ai/laws.dm

```diff
@@ -21,6 +21,7 @@
 		for(var/mob/living/silicon/robot/R in connected_robots)
 			if(R.lawupdate)
 				R.lawsync()

```

Comment:
Tbh, I really prefer supermichael's solution to this silicon law issue. Adding a second argument for this is awkward.